src/styles/cryptolist.css

```diff
@@ -0,0 +1,82 @@
+.coinCard h1,
+.coinCard h2,
+.coinCard h3 {
+  margin: 0;
+  padding: 0;
+  letter-spacing: 0.04em;
+  font-family: "Gill Sans", "Lato", sans-serif;
+  color: #000;
+}
+
+.searchContainer {
+  text-align: center;
+  padding: 50px;
+  background-color: #3a3a3a;
+}
+
+.searchBar {
+  padding: 15px;
+  width: 75%;
+  border: 1px solid #000;
+  outline: none;
+  border-radius: 4px;
+}
+
+.searchBar::placeholder {
+  font-size: medium;
+  font-weight: 300;
+  letter-spacing: 0.1em;
+}
+
+.searchBtn {
+  padding: 15px 20px;
+  background-color: #9b980d;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.coinsList {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.coinCard {
+  width: 50%;
+  background-color: #b24364;
+  border: 1px solid #fff;
+  padding: 60px 60px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
- [ ] Great job on styling your project so beautifully! 👏 However, please note that the project requirements specifically state that you should personalize the project design while maintaining its `color`, `layout`, and `Typography`. 

![Screenshot](https://github.com/GhausAliKhan/cry-for-crypto/assets/107192076/adace109-30c6-456d-9621-07df22a85a17)

   - I can see that you've done a fantastic job maintaining the design `layout` and `Typography`, but it looks like the `color` needs some alignment with the project's requirements. To fully adhere to the project's design specifications, please make sure to select one of the colors provided in the design and use it consistently in your project styles. Here's a reference to the available colors you can choose from:

![Color Palette](https://github.com/GhausAliKhan/cry-for-crypto/assets/107192076/3aeab073-da55-4a15-b374-500ba6f9a664)

Keep up the excellent work! 🎨✨

Reply:
Thankyou @Strangeal  for your clear instructions. I've added the changes 💯